applications/appunorganized/appunorganized-plugins/org.csstudio.service.jdbcsample/META-INF/MANIFEST.MF

```diff
@@ -5,8 +5,9 @@ Bundle-SymbolicName: org.csstudio.service.jdbcsample;singleton:=true
 Bundle-Version: 3.2.0.qualifier
 Bundle-Activator: org.csstudio.service.jdbcsample.Activator
 Require-Bundle: org.eclipse.core.runtime,
- org.epics.vtype,
- org.csstudio.utility.pvmanager,
- org.csstudio.utility.pvmanager.jdbc
+ org.diirt.vtype,
```

Comment:
@shroffk jdbcsample service, I made these changes, although it wasn't correct.  This plugin is not in a feature, so I removed it from the build